dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+dcos-test-utils==0.1
```

Comment:
`dev-requirements` are only ever installed *with* `requirements`. Therefore there shouldn't be anything here that is required for the library to be used.